src/components/Authentication/authentication.css

```diff
@@ -0,0 +1,61 @@
+.authentication-container{
+	background-color: var(--bg-color-primary);
+	height:fit-content;
+	width: 50em;
```

Comment:
Suggestion: You could use rems here for consistency. Please let me know if you are using it for a case

Reply:
This was a mistake. Thank you for pointing it out.